### PR TITLE
[Addon] metrics addons

### PR DESCRIPTION
**Pull request by Somefive**
### Description of your changes

1. Modify `xxx: *"" | string` to `xxx?: string` to fit velaux openapi usage.
2. Move parameter.name to const.name to fix velaux openapi usage.
3. Fix Grafana KubeVela-System Dashboard not work when thanos is not installed.

### Checklist

I have:

- [x] Title of the PR starts with type (e.g. `[Addon]` , `[example]` or `[Doc]`).
- [ ] Updated/Added any relevant [documentation](https://kubevela.io/docs/reference/addons/overview) and [examples](https://github.com/kubevela/catalog/tree/master/examples).
- [ ] New addon should be put in [experimental](https://github.com/kubevela/catalog/tree/master/experimental/addons).
- [ ] Update addon should modify the `version` in `metadata.yaml` to generate a new version.

### CODE REVIEW

File: addons/prometheus-server/parameter.cue [hunk 1]
```diff
@@ -24,9 +27,11 @@ parameter: {
 	// +usage=Specify the service type for expose prometheus server. If empty, it will be not exposed.
 	serviceType: *"ClusterIP" | "NodePort" | "LoadBalancer" | ""
 	// +usage=Specify the storage size to use. If empty, emptyDir will be used. Otherwise pvc will be used.
-	storage: *"" | string
+	storage?: string
```
Comment (barnettZQG) [inline addons/prometheus-server/parameter.cue:30]: What happens if the storage is `abc`?

Comment (Somefive) [reply addons/prometheus-server/parameter.cue:30]: Kubernetes apiserver will reject the resource creation request.

Comment (barnettZQG) [reply addons/prometheus-server/parameter.cue:30]: Maybe this is more clear:
```cue
storage: {
   type: *"EmptyDir" | "PVC" 
   size?: int
}
```

Comment (Somefive) [reply addons/prometheus-server/parameter.cue:30]: Do we need to change type from "PVC" to "PersistentVolumeClaim"?

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**